code/bml_loader.js

```diff
@@ -0,0 +1,20 @@
+function loadBookmarklet(kbdid, langid) {
```

Comment:
Let's call this file `bookmarklet-load.js` rather than `bml_loader.js`. It's marginally longer but it's clearer what it is :grin:

Next, this function is a global across all websites where this is injected, so we should make it a very unique name, e.g. `loadKeymanWebBookmarklet`